content/README.md

```diff
@@ -26,4 +26,4 @@ When the module is compiled (using `dotnet build`) it's configured to generate a
 
 ## Notes
 
-This module was created using `v1.3.0` of [Etch.OrchardCore.ModuleBoilerplate](https://github.com/EtchUK/Etch.OrchardCore.ModuleBoilerplate) template.
+This module was created using `v1.4.0` of [Etch.OrchardCore.ModuleBoilerplate](https://github.com/EtchUK/Etch.OrchardCore.ModuleBoilerplate) template.
```

Comment:
This needs to use the `$(VersionNumber)` as well, otherwise we'll have to manually update this each time which is what we're trying to move away from. The pipeline will need to be updated to target this file when doing the find/replace.